arrays/c-or-cpp/Tint.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream.h>

```

Comment:
Please rename the file from `Tint.cpp` to `data-before-sort.cpp`, and fix the ident of file

Reply:
done